patches/@php-wasm+universal+0.9.44.patch

```diff
@@ -1,10 +0,0 @@
-diff --git a/node_modules/@php-wasm/universal/index.cjs b/node_modules/@php-wasm/universal/index.cjs
```

Comment:
We are removing the patch that was added in https://github.com/Automattic/studio/pull/532 and setting the `cookieStore` to `false` as we don't want to use Playground cookie store for Playground running from CLI.  This allows the browser to manage the cookies.